src/app/exp-feature-page/flask.svg

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
That is only in the .svg file. It is not in the .html. It is part of the xml spec for xml based files, svg is an xml type of file.